assemblies/features/src/main/resources/features.xml

```diff
@@ -55,7 +55,7 @@
     <bundle>mvn:pentaho/pentaho-hadoop-shims-mapr-osgi-jaas/${project.version}</bundle>
   </feature>
   <feature name="pentaho-big-data-kettle-plugins" version="1.0">
-    <feature>pentaho-big-data-plugin-api</feature>
+    <feature>pentaho-big-data-plugin-shim-impl</feature>
```

Comment:
I agree this your assessment @hudak . I will try separate out the bundle that needed shim common and see how that looks